criu/cr-check.c

```diff
@@ -1589,6 +1589,13 @@ static int check_overlayfs_maps(void)
 	return status == 0 ? 0 : -1;
 }
 
+static int check_pagemap_scan_guard_pages(void)
+{
+	kerndat_warn_about_madv_guards();
```

Comment:
nit: cr_check is always called after kerndat_init, so we don't need to call kerndat_warn_about_madv_guards.